Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -69,6 +74,16 @@ - (instancetype)initWithOptions:(SentryOptions *)options
 
 - (void)start
 {
+    for (NSString *include in self.inAppLogic.inAppIncludes) {
+        NSString *pathToImage = [self.binaryImageCache pathForImage:include];
+        if (pathToImage != nil) {
+            [self swizzleUIViewControllersOfImage:pathToImage];
+        } else {
+            SENTRY_LOG_DEBUG(
+                @"UIViewControllerSwizzling: Failed to find binary image for %@ module.", include);
```

Comment:
`l`: I think this should be `SENTRY_LOG_WARN`. You don't need to add `UIViewControllerSwizzling` to the log message, as every log message includes the class and the line number. We could make the message more descriptive so the user knows what is going on: `Failed to find the binary image for inAppInclude <@%> and, therefore can't instrument UIViewControllers in that binary.`